cmd/controller/app/options/options.go

```diff
@@ -83,6 +85,7 @@ const (
 	defaultTLSACMEIssuerKind           = "Issuer"
 	defaultACMEIssuerChallengeType     = "http01"
 	defaultACMEIssuerDNS01ProviderName = ""
+	defaultEnableCertificateOwnerRef   = true
```

Comment:
/cc @kragniz 

What do you think about defaulting this to true? 👍 or 👎? This will mean that Secret resources will be deleted if the corresponding Certificate goes away.

I am beginning to think defaulting this to `false` will help prevent any unexpected surprises (i.e. someone deletes the Certificate CRD type, causing all Certificate resources to go away and thus all Secret resources to be cleaned up, bringing down their production infrastructure 🙃)

Reply:
@munnerz @kragniz I set the default value to `false`.